crux-mir/README.md

```diff
@@ -102,13 +137,62 @@ To compile and test a single Rust program:
 
 (Should print 2.)
 
+### Code coverage
+
+`crux-mir` has a code coverage tool that reports branches where only one side or the other was explored during symbolic execution. Note that due to how `mir-json` works, this will only report issues in functions that are called at least once from a `#[crux::test]` entrypoint. If you have dead code (i.e., functions that are never called by any entrypoint), then `mir-json` will discard the dead functions, which means that they won't be mentioned at all in the coverage output. Code coverage currently does not report the percentage of the coverage, only lists the paths that are not covered.
+
+* Clone this directory with `git clone https://github.com/GaloisInc/crucible.git` as we will need the `report-coverage` utility tool
+* In your crate run:
+  ```
+  $ crux-mir --lib --  --branch-coverage --path-sat --output-directory test-coverage
+  ```
+* In your crate, run the following command, pointing it towards the `report-coverage` folder in `crucible` directory (note that you need to point to a directory in `test-coverage` folder that contains `report_data.js`, and the path is currently rather mangled):
+  ```
+  $ cargo run --manifest-path $PATH_TO_CRUCIBLE_REPO/report-coverage/Cargo.toml -- test-coverage/test/62f2dedb\:\:f\[0\]/report_data.js
+  ```
+* This will report all paths not covered, including ones from the standard library, for example:
+  ```
+  warning: branch condition never has value true
+    ┌─ ./libs/core/src/ub_checks.rs:1:1
+    │
+  1 │ 
+    │ ^
+    │
+    ┌─ ./libs/core/src/num/mod.rs:1:1
+    │
+  1 │ 
+    │ - in this macro invocation
+
+  warning: branch condition never has value true
+    ┌─ ./libs/core/src/ub_checks.rs:1:1
+    │
+  1 │ 
+    │ ^
+
+  warning: branch condition never has value false
+    ┌─ lib.rs:43:13
+    │
+  43 │     assert!(a == b);
+    │             ^^^^^^
+
+  ```
+* To limit the coverage only to the code in your crate, use `--filter` to point the tool to the file you want to analyze. Then you get a more condensed output, for example:
```

Comment:
Please include a complete invocation of the coverage script using `--filter` to make it clearer how to use this option.

Reply:
Done